site/src/load.rs

```diff
@@ -127,7 +135,7 @@ pub struct SiteCtxt {
     /// Index of various common queries
     pub index: ArcSwap<crate::db::Index>,
     /// Cached master-branch Rust commits
-    pub master_commits: ArcSwap<MasterCommitCache>,
+    pub master_commits: Arc<ArcSwap<MasterCommitCache>>, // outer Arc enables mutation in background task
```

Comment:
This was necessary to be able to access the cache in a background task. I think it makes sense, but please let me know if there's a better way.